docs/FAQ.md

```diff
@@ -563,6 +561,31 @@ Usually, the different commits associated with the conflicted bookmark should al
 appear in the log, but if they don't you can use `jj bookmark list`to show all the
 commits associated with it.
 
+### How can I avoid accidentally tracking new files I didn't mean to track?
+
+Users with habits from a different VCS like Git may be used to new files being explicitly tracked.
```

Comment:
"new files being explicitly tracked" -> "new files needing to be explicitly tracked"?